package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.3.0",
   "description": "LoopBack Connector for IBM Cloudant DB",
   "engines": {
-    "node": ">=6"
+    "node": ">=8"
```

Comment:
Let's use `8.9` please. Older 8.x versions of Node are missing few useful features we are depending on in LB4.

Reply:
Never mind, I see you have discussed this aspect, I am fine with your decision.